Sources/BraveWallet/Crypto/Transactions/TransactionParser.swift

```diff
@@ -543,6 +554,10 @@ struct Eth721TransferDetails: Equatable {
   let tokenId: String
 }
 
+struct SolanaDappTxDetails: Equatable {
+  
+}
```

Comment:
empty struct?

Reply:
Removed in [542fba8](https://github.com/brave/brave-ios/pull/6050/commits/542fba84d52d2bee4972af100d529aeef7d6532c). Will be re-added (with properties) in PR to support this transaction type.